mono/mini/mini-unwind.h

```diff
@@ -177,7 +177,7 @@ mono_unwind_ops_encode_full (GSList *unwind_ops, guint32 *out_len, gboolean enab
 guint8*
 mono_unwind_ops_encode (GSList *unwind_ops, guint32 *out_len);
 
-void
+gboolean
 mono_unwind_frame (guint8 *unwind_info, guint32 unwind_info_len, 
```

Comment:
It would be tempting to use `MonoError` here, but I suppose it's not async safe, and this functions needs to?

Reply:
MonoError is async safe as long as you don't use certain error paths.

Reply:
You asked me to leave this not-MonoError in our call, commenting here so I can remember in the future